code/sound/voicerec.cpp

```diff
@@ -51,6 +54,27 @@ extern int Msg_instance;;
 extern int Msg_shortcut_command;
 extern int Squad_msg_mode;
 
+namespace
+{
+	bool system_event_handler(const SDL_Event& e)
+	{
+		switch (e.syswm.msg->msg.win.msg)
+		{
+		case WM_RECOEVENT:
+			if (Game_mode & GM_IN_MISSION && Cmdline_voice_recognition)
+			{
+				VOICEREC_process_event(e.syswm.msg->msg.win.hwnd);
+				return true;
+			}
+			break;
```

Comment:
That wouldn't satisfy the requirements of an event handler. It's only supposed to return `true` if the event was actually handled which is not the case when `VOICEREC_process_event` wasn't called.